src/views/compute/wizard/TemplateIsoSelection.vue

```diff
@@ -113,6 +112,7 @@ export default {
     inputDecorator (newValue, oldValue) {
       if (newValue !== oldValue) {
         this.filter = ''
+        this.filterType = 'featured'
```

Comment:
@utchoang Can we keep this too so that context is not lost switching between Templates & ISOs